opampserver/pkg/sdkconfig/instrumentationconfigs_controller.go

```diff
@@ -24,7 +24,7 @@ func (i *InstrumentationConfigReconciler) Reconcile(ctx context.Context, req ctr
 
 	if err != nil {
 		if apierrors.IsNotFound(err) {
-			instrumentationConfig = nil
+			return ctrl.Result{}, nil
 		} else {
 			return ctrl.Result{}, err
 		}
```

Comment:
Thanks!

If you like, this can be refactored to:

```golang
return ctrl.Result{}, client.IgnoreNotFound(err)
```

Reply:
nice, thanks